pkg/resources/deviceSelectors.go

```diff
@@ -171,11 +173,20 @@ func contains(hay []string, needle string) bool {
 	return false
 }
 
-func getItem(hay []string, needle string) string {
+func getItem(hay []string, needle string, what string) string {
 	for _, item := range hay {
+		// May be an exact match or a regex, try both.
```

Comment:
I think I'd like to have this regex matching an optional feature that would be off by default. I am worried a PF name could end up being a regex expression and cause strange matches to occur.

Reply:
With the re{}, intent would be clear that it must compile/match.